src/main/resources/reference.conf

```diff
@@ -224,6 +224,41 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  google-project = {
+    actionPatternObjects = {
+      list_notebook_cluster = {
+        description = "list all notebook clusters in this billing-project"
+      }
+      launch_notebook_cluster = {
+        description = "launch a new notebook cluster in this billing-project"
+      }
+      delete_notebook_cluster = {
+        description = ""
+      }
+      stop_start_notebook_cluster = {
+        description = "stop and start notebook clusters in this billing-project"
+      }
+      list_persistent_disk = {
+        description = "list all persistent disks in this billing-project"
+      }
+      create_persistent_disk = {
+        description = "create persistent disk in this billing-project"
+      }
+      delete_persistent_disk = {
+        description = "delete persistent disk in this billing-project"
+      }
+      create_kubernetes_app = {
+        description = "create kubernetes application in this billing-project"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
```

Comment:
I believe we also want the `notebook-user` role to be copied onto this resource type from `billing-project`. I think we don't need to bother with `batch-compute-user` since we're removing the one check that I know about. I leave it up to Doug to confirm whether or not that's true.